e2e/test/scenarios/embedding/embedding-theme-editor/theme-listing.cy.spec.ts

```diff
@@ -0,0 +1,36 @@
+const { H } = cy;
+
+describe("scenarios > embedding > themes > theme listing", () => {
+  beforeEach(() => {
+    H.restore();
+    cy.signInAsAdmin();
+  });
+
+  it("shows empty state and allows creating a new theme", () => {
+    cy.visit("/admin/embedding/themes");
+
+    cy.log("theme is visible in embedding sidebar");
+    cy.findByTestId("admin-layout-sidebar")
+      .findByText("Themes")
+      .should("be.visible");
+
+    H.main().within(() => {
+      cy.log("empty state is visible");
+      cy.findByText("Create your first theme to get started").should(
+        "be.visible",
+      );
+
+      cy.log("create a theme");
+      cy.findByRole("button", { name: /New theme/ }).click();
+
+      // TODO(EMB-946): assert that it navigates to the theme editor page.
```

Comment:
We don't have the theme editor page yet. Once we have it in EMB-946, we'll do the assertion on the theme editor page's content instead. For now, we just check that a card is created.